registry/handlers/hooks.go

```diff
@@ -0,0 +1,35 @@
+package handlers
+
+import (
+	"github.com/Sirupsen/logrus"
+)
+
+// LogHook is for hooking Panic in web application
+type LogHook struct {
+	LevelsParam []string
+	Type        string
+	MailParam   Mail
+}
+
+// Fire forwards an error to LogHook
+func (hook *LogHook) Fire(entry *logrus.Entry) error {
+	switch hook.Type {
+	case "email":
+		MailSending(hook.MailParam,
```

Comment:
The error return by MailSending is not caught here.  Can you log the error if it's !nil?